### PR TITLE
Device style fixes

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] you can set the startup tab / routing from the environment var now. example:
```
KEYBASE_LOCAL_DEBUG_JSON="{\"overrideLoggedInTab\":\"tabs:devicesTab\", \"initialTabState\": {\"tabs:devicesTab\": []}}"
```
- [x] you can set the main window start position. to get the current position in the debugger
```
require('electron').remote.getCurrentWindow().getPosition()
```
then
```
KEYBASE_LOCAL_DEBUG_JSON="{\"overrideLoggedInTab\":\"tabs:devicesTab\", \"initialTabState\": {\"tabs:devicesTab\": []}, \"forceMainWindowPosition\": {\"x\": 40, \"y\": -1400}}"
```
- [x] back button height fixed. 
- [x] don't make revoke button take up the whole row. 
- [x] make ignore caret go to the right not up. 
- [x] make row clickable styled. 
- [x] make whole row clickable
- [x] Top bar should be 64px (in general we switched to a 8px-increment spacing system)
- [x] Font-size for device names should be 16px (currently 18px)
- [x] Icons should be the 48x48 ones and be 8px away from the left
- [x] Text should be 16px away from the icons
- [x] Revoke link should be one size down: 14px (currently 16)
- [x] Add new link should be 8px away from the icon (currently too far on the right)
- [x] "Revoked devices" section label should be 14px and black_60
- [x] Little caret when collapsed should be looking to the right, not to the top (specs for this here: https://zpl.io/TyTsH)
- [x] Background should be lightGrey, not white. Only the list items and the top bar are white.
- [x] Cursor should be the little hand when hovering on this section divider since it's clickable
- [x] "Revoked devices will no longer..." message should be Body small (14px regular). (Currently 16px Semibold)
- [x] Content should vertically center on the confirm screens when revoking a device.
- [x] Little caret looks blurry - are we using the base-8px one from the font? (ping Marco)

@MarcoPolo lets talk about the icon font stuff. the caret at 8px looks incorrect. not sure if i'm thinking about this incorrectly

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 8ca7e6037d55...c79c668c050a introduced some visual changes:
 * Changed: **Devices_List-Devices** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466780609950-8ca7e6037d55-c79c668c050a/Devices_List-Devices-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466780609950-8ca7e6037d55-c79c668c050a/Devices_List-Devices-c79c668c050a.png)  
   Devices_List-Devices comparison
 * Changed: **Device_Page-Unlock** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466780609950-8ca7e6037d55-c79c668c050a/Device_Page-Unlock-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466780609950-8ca7e6037d55-c79c668c050a/Device_Page-Unlock-c79c668c050a.png)  
   Device_Page-Unlock comparison
 * Changed: **Device_Page-Revoked** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466780609950-8ca7e6037d55-c79c668c050a/Device_Page-Revoked-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466780609950-8ca7e6037d55-c79c668c050a/Device_Page-Revoked-c79c668c050a.png)  
   Device_Page-Revoked comparison
 * Changed: **Device_Page-Paper** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466780609950-8ca7e6037d55-c79c668c050a/Device_Page-Paper-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466780609950-8ca7e6037d55-c79c668c050a/Device_Page-Paper-c79c668c050a.png)  
   Device_Page-Paper comparison
 * Changed: **Device_Page-Normal** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466780609950-8ca7e6037d55-c79c668c050a/Device_Page-Normal-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466780609950-8ca7e6037d55-c79c668c050a/Device_Page-Normal-c79c668c050a.png)  
   Device_Page-Normal comparison

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 40191f81f485...38adfa269cb0 introduced some visual changes on win32:
 * Changed: **Folders_Clear_History_Confirmation-Normal_Public** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466780547536-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Public-40191f81f485.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466780547536-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Public-38adfa269cb0.png)  
   Folders_Clear_History_Confirmation-Normal_Public comparison
 * Changed: **Folders_Clear_History_Confirmation-Normal_Private** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466780547536-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Private-40191f81f485.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466780547536-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Private-38adfa269cb0.png)  
   Folders_Clear_History_Confirmation-Normal_Private comparison
 * Changed: **Folders_Clear_History_Confirmation-Long_Name_Public** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466780547536-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Long_Name_Public-40191f81f485.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466780547536-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Long_Name_Public-38adfa269cb0.png)  
   Folders_Clear_History_Confirmation-Long_Name_Public comparison

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 8ca7e6037d55...7ca8081658ae introduced some visual changes:
 * Changed: **Devices_List-Devices** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466782680792-8ca7e6037d55-7ca8081658ae/Devices_List-Devices-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466782680792-8ca7e6037d55-7ca8081658ae/Devices_List-Devices-7ca8081658ae.png)  
   Devices_List-Devices comparison
 * Changed: **Device_Page-Unlock** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466782680792-8ca7e6037d55-7ca8081658ae/Device_Page-Unlock-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466782680792-8ca7e6037d55-7ca8081658ae/Device_Page-Unlock-7ca8081658ae.png)  
   Device_Page-Unlock comparison
 * Changed: **Device_Page-Revoked** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466782680792-8ca7e6037d55-7ca8081658ae/Device_Page-Revoked-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466782680792-8ca7e6037d55-7ca8081658ae/Device_Page-Revoked-7ca8081658ae.png)  
   Device_Page-Revoked comparison
 * Changed: **Device_Page-Paper** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466782680792-8ca7e6037d55-7ca8081658ae/Device_Page-Paper-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466782680792-8ca7e6037d55-7ca8081658ae/Device_Page-Paper-7ca8081658ae.png)  
   Device_Page-Paper comparison
 * Changed: **Device_Page-Normal** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466782680792-8ca7e6037d55-7ca8081658ae/Device_Page-Normal-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466782680792-8ca7e6037d55-7ca8081658ae/Device_Page-Normal-7ca8081658ae.png)  
   Device_Page-Normal comparison

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 40191f81f485...38adfa269cb0 introduced some visual changes on win32:
 * Changed: **Folders_Clear_History_Confirmation-Normal_Public** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466782609249-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Public-40191f81f485.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466782609249-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Public-38adfa269cb0.png)  
   Folders_Clear_History_Confirmation-Normal_Public comparison
 * Changed: **Folders_Clear_History_Confirmation-Normal_Private** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466782609249-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Private-40191f81f485.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466782609249-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Private-38adfa269cb0.png)  
   Folders_Clear_History_Confirmation-Normal_Private comparison
 * Changed: **Folders_Clear_History_Confirmation-Long_Name_Public** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466782609249-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Long_Name_Public-40191f81f485.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466782609249-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Long_Name_Public-38adfa269cb0.png)  
   Folders_Clear_History_Confirmation-Long_Name_Public comparison

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 8ca7e6037d55...d2ba6fc5cb7b introduced some visual changes:
 * Changed: **Devices_List-Devices** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466783530870-8ca7e6037d55-d2ba6fc5cb7b/Devices_List-Devices-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466783530870-8ca7e6037d55-d2ba6fc5cb7b/Devices_List-Devices-d2ba6fc5cb7b.png)  
   Devices_List-Devices comparison
 * Changed: **Device_Page-Unlock** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466783530870-8ca7e6037d55-d2ba6fc5cb7b/Device_Page-Unlock-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466783530870-8ca7e6037d55-d2ba6fc5cb7b/Device_Page-Unlock-d2ba6fc5cb7b.png)  
   Device_Page-Unlock comparison
 * Changed: **Device_Page-Revoked** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466783530870-8ca7e6037d55-d2ba6fc5cb7b/Device_Page-Revoked-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466783530870-8ca7e6037d55-d2ba6fc5cb7b/Device_Page-Revoked-d2ba6fc5cb7b.png)  
   Device_Page-Revoked comparison
 * Changed: **Device_Page-Paper** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466783530870-8ca7e6037d55-d2ba6fc5cb7b/Device_Page-Paper-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466783530870-8ca7e6037d55-d2ba6fc5cb7b/Device_Page-Paper-d2ba6fc5cb7b.png)  
   Device_Page-Paper comparison
 * Changed: **Device_Page-Normal** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466783530870-8ca7e6037d55-d2ba6fc5cb7b/Device_Page-Normal-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466783530870-8ca7e6037d55-d2ba6fc5cb7b/Device_Page-Normal-d2ba6fc5cb7b.png)  
   Device_Page-Normal comparison

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 40191f81f485...38adfa269cb0 introduced some visual changes on win32:
 * Changed: **Folders_Clear_History_Confirmation-Normal_Public** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466783465119-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Public-40191f81f485.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466783465119-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Public-38adfa269cb0.png)  
   Folders_Clear_History_Confirmation-Normal_Public comparison
 * Changed: **Folders_Clear_History_Confirmation-Normal_Private** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466783465119-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Private-40191f81f485.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466783465119-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Private-38adfa269cb0.png)  
   Folders_Clear_History_Confirmation-Normal_Private comparison
 * Changed: **Folders_Clear_History_Confirmation-Long_Name_Public** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466783465119-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Long_Name_Public-40191f81f485.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466783465119-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Long_Name_Public-38adfa269cb0.png)  
   Folders_Clear_History_Confirmation-Long_Name_Public comparison

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 8ca7e6037d55...daea91450a3e introduced some visual changes:
 * Changed: **Devices_List-Devices** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466783768767-8ca7e6037d55-daea91450a3e/Devices_List-Devices-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466783768767-8ca7e6037d55-daea91450a3e/Devices_List-Devices-daea91450a3e.png)  
   Devices_List-Devices comparison
 * Changed: **Device_Page-Unlock** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466783768767-8ca7e6037d55-daea91450a3e/Device_Page-Unlock-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466783768767-8ca7e6037d55-daea91450a3e/Device_Page-Unlock-daea91450a3e.png)  
   Device_Page-Unlock comparison
 * Changed: **Device_Page-Revoked** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466783768767-8ca7e6037d55-daea91450a3e/Device_Page-Revoked-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466783768767-8ca7e6037d55-daea91450a3e/Device_Page-Revoked-daea91450a3e.png)  
   Device_Page-Revoked comparison
 * Changed: **Device_Page-Paper** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466783768767-8ca7e6037d55-daea91450a3e/Device_Page-Paper-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466783768767-8ca7e6037d55-daea91450a3e/Device_Page-Paper-daea91450a3e.png)  
   Device_Page-Paper comparison
 * Changed: **Device_Page-Normal** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466783768767-8ca7e6037d55-daea91450a3e/Device_Page-Normal-8ca7e6037d55.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466783768767-8ca7e6037d55-daea91450a3e/Device_Page-Normal-daea91450a3e.png)  
   Device_Page-Normal comparison

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 40191f81f485...38adfa269cb0 introduced some visual changes on win32:
 * Changed: **Folders_Clear_History_Confirmation-Normal_Public** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466783748004-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Public-40191f81f485.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466783748004-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Public-38adfa269cb0.png)  
   Folders_Clear_History_Confirmation-Normal_Public comparison
 * Changed: **Folders_Clear_History_Confirmation-Normal_Private** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466783748004-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Private-40191f81f485.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466783748004-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Private-38adfa269cb0.png)  
   Folders_Clear_History_Confirmation-Normal_Private comparison
 * Changed: **Folders_Clear_History_Confirmation-Long_Name_Public** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466783748004-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Long_Name_Public-40191f81f485.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466783748004-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Long_Name_Public-38adfa269cb0.png)  
   Folders_Clear_History_Confirmation-Long_Name_Public comparison

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready to look at

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 40191f81f485...38adfa269cb0 introduced some visual changes on win32:
 * Changed: **Folders_Clear_History_Confirmation-Normal_Public** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466784628915-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Public-40191f81f485.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466784628915-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Public-38adfa269cb0.png)  
   Folders_Clear_History_Confirmation-Normal_Public comparison
 * Changed: **Folders_Clear_History_Confirmation-Normal_Private** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466784628915-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Private-40191f81f485.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466784628915-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Normal_Private-38adfa269cb0.png)  
   Folders_Clear_History_Confirmation-Normal_Private comparison
 * Changed: **Folders_Clear_History_Confirmation-Long_Name_Public** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466784628915-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Long_Name_Public-40191f81f485.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466784628915-40191f81f485-38adfa269cb0-win32/Folders_Clear_History_Confirmation-Long_Name_Public-38adfa269cb0.png)  
   Folders_Clear_History_Confirmation-Long_Name_Public comparison

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 272d7b878887...0789a066735d introduced some visual changes:
 * Changed: **Devices_List-Devices** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466784734212-272d7b878887-0789a066735d/Devices_List-Devices-272d7b878887.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466784734212-272d7b878887-0789a066735d/Devices_List-Devices-0789a066735d.png)  
   Devices_List-Devices comparison
 * Changed: **Device_Page-Unlock** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466784734212-272d7b878887-0789a066735d/Device_Page-Unlock-272d7b878887.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466784734212-272d7b878887-0789a066735d/Device_Page-Unlock-0789a066735d.png)  
   Device_Page-Unlock comparison
 * Changed: **Device_Page-Revoked** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466784734212-272d7b878887-0789a066735d/Device_Page-Revoked-272d7b878887.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466784734212-272d7b878887-0789a066735d/Device_Page-Revoked-0789a066735d.png)  
   Device_Page-Revoked comparison
 * Changed: **Device_Page-Paper** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466784734212-272d7b878887-0789a066735d/Device_Page-Paper-272d7b878887.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466784734212-272d7b878887-0789a066735d/Device_Page-Paper-0789a066735d.png)  
   Device_Page-Paper comparison
 * Changed: **Device_Page-Normal** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466784734212-272d7b878887-0789a066735d/Device_Page-Normal-272d7b878887.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466784734212-272d7b878887-0789a066735d/Device_Page-Normal-0789a066735d.png)  
   Device_Page-Normal comparison

File: shared/devices/render.desktop.js [hunk 1]
```diff
@@ -56,28 +56,32 @@ const DeviceRow = ({device, revoked, showRemoveDevicePage, showExistingDevicePag
...
       className='existing-device-container'
       key={device.name}
+      onClick={() => showExistingDevicePage(device)}
       style={{...stylesCommonRow, backgroundColor: revoked ? globalColors.white_40 : globalColors.white}}>
-      <Box style={revoked ? stylesRevokedIconColumn : stylesIconColumn}>
+      <Box style={revoked ? {opacity: 0.2} : {}}>
         <Icon type={icon} />
       </Box>
-      <Box style={stylesCommonColumn} onClick={() => showExistingDevicePage(device)}>
+      <Box style={{flex: 1, marginLeft: 16}}>
```
Comment (chromakode) [inline shared/devices/render.desktop.js]: `16` -> `globalMargins.small`?

File: shared/devices/render.desktop.js [hunk 2]
```diff
@@ -89,13 +93,9 @@ const RevokedDevices = ({revokedDevices, showExistingDevicePage}) => (
...
-  <Box style={{...stylesCommonRow, ...globalStyles.clickable, backgroundColor: globalColors.white}} onClick={addNewDevice}>
-    <Box style={stylesCommonColumn}>
-      <Icon type='devices-add-s' />
-    </Box>
-    <Box style={stylesCommonColumn}>
-      <Text type='BodyPrimaryLink' onClick={addNewDevice}>Add new...</Text>
-    </Box>
+  <Box style={{...stylesCommonRow, ...globalStyles.clickable, backgroundColor: globalColors.white, height: globalMargins.xlarge}} onClick={addNewDevice}>
+    <Icon type='icon-devices-add-64-x-48' />
+    <Text type='BodyPrimaryLink' onClick={addNewDevice} style={{marginLeft: 8}}>Add new...</Text>
```
Comment (chromakode) [inline shared/devices/render.desktop.js]: `8` -> `globalMargins.tiny`?

**Comment by chromakode** [comment]
:+1: % nits

File: shared/devices/render.desktop.js [hunk 3]
```diff
@@ -23,11 +23,11 @@ class RevokedHeader extends Component<void, RevokedHeaderProps, RevokedHeaderSta
   }
 
   render () {
-    const iconType = this.state.expanded ? 'fa-caret-down' : 'fa-caret-up'
+    const iconType = this.state.expanded ? 'fa-caret-down' : 'fa-caret-right'
```
Comment (MarcoPolo) [inline shared/devices/render.desktop.js]: You want to use our own pixel perfect carets:
from icon.js.flow
```
...
    | 'fa-kb-iconfont-caret-down'
    | 'fa-kb-iconfont-caret-right'
```

**Comment by MarcoPolo** [comment]
👍

**Comment by cjb** [comment]
retest this please

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 11e6f1038439...ebcd92c1936e introduced some visual changes:
 * Changed: **Devices_List-Devices** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466988157682-11e6f1038439-ebcd92c1936e/Devices_List-Devices-11e6f1038439.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466988157682-11e6f1038439-ebcd92c1936e/Devices_List-Devices-ebcd92c1936e.png)  
   Devices_List-Devices comparison
 * Changed: **Device_Page-Unlock** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466988157682-11e6f1038439-ebcd92c1936e/Device_Page-Unlock-11e6f1038439.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466988157682-11e6f1038439-ebcd92c1936e/Device_Page-Unlock-ebcd92c1936e.png)  
   Device_Page-Unlock comparison
 * Changed: **Device_Page-Revoked** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466988157682-11e6f1038439-ebcd92c1936e/Device_Page-Revoked-11e6f1038439.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466988157682-11e6f1038439-ebcd92c1936e/Device_Page-Revoked-ebcd92c1936e.png)  
   Device_Page-Revoked comparison
 * Changed: **Device_Page-Paper** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466988157682-11e6f1038439-ebcd92c1936e/Device_Page-Paper-11e6f1038439.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466988157682-11e6f1038439-ebcd92c1936e/Device_Page-Paper-ebcd92c1936e.png)  
   Device_Page-Paper comparison
 * Changed: **Device_Page-Normal** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466988157682-11e6f1038439-ebcd92c1936e/Device_Page-Normal-11e6f1038439.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466988157682-11e6f1038439-ebcd92c1936e/Device_Page-Normal-ebcd92c1936e.png)  
   Device_Page-Normal comparison

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits cf3aa14878a2...15cedf955a78 introduced some visual changes:
 * Changed: **Devices_List-Devices** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1467036093822-cf3aa14878a2-15cedf955a78/Devices_List-Devices-cf3aa14878a2.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1467036093822-cf3aa14878a2-15cedf955a78/Devices_List-Devices-15cedf955a78.png)  
   Devices_List-Devices comparison
 * Changed: **Device_Page-Unlock** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1467036093822-cf3aa14878a2-15cedf955a78/Device_Page-Unlock-cf3aa14878a2.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1467036093822-cf3aa14878a2-15cedf955a78/Device_Page-Unlock-15cedf955a78.png)  
   Device_Page-Unlock comparison
 * Changed: **Device_Page-Revoked** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1467036093822-cf3aa14878a2-15cedf955a78/Device_Page-Revoked-cf3aa14878a2.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1467036093822-cf3aa14878a2-15cedf955a78/Device_Page-Revoked-15cedf955a78.png)  
   Device_Page-Revoked comparison
 * Changed: **Device_Page-Paper** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1467036093822-cf3aa14878a2-15cedf955a78/Device_Page-Paper-cf3aa14878a2.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1467036093822-cf3aa14878a2-15cedf955a78/Device_Page-Paper-15cedf955a78.png)  
   Device_Page-Paper comparison
 * Changed: **Device_Page-Normal** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1467036093822-cf3aa14878a2-15cedf955a78/Device_Page-Normal-cf3aa14878a2.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1467036093822-cf3aa14878a2-15cedf955a78/Device_Page-Normal-15cedf955a78.png)  
   Device_Page-Normal comparison

**Pull request merged**